netlify.toml

```diff
@@ -0,0 +1,4 @@
+[[redirect]]
+  from = "docs/accessibility-assessment-ial2-identity-verification-process.pdf"
+  to = "docs/accessibility-assessment-identity-verification-process.pdf"
+  force = true
```

Comment:
Netlify isn't part of the build process, does Jekyll know how to process this?

Reply:
I think that doing redirects for non-HTML URLs in Jekyll is really tough, I think it would be easier to replace the file than to rename it

Reply:
Is it possible to use `redirect_to` for this? Not super elegant, but maybe we can pretend it's a PDF with a file path like `docs/accessibility-assessment-identity-verification-process.pdf/index.md` 😬 ?

Example `redirect_to`: https://github.com/18F/identity-site/blob/main/content/_developers/index._en.md?plain=1

Reply:
I have not been able to get `redirect_to` on a that and not even with the `index` trick either, I don't think